### ISSUE TITLE
Provide more info

### ISSUE THREAD

**Issue by kaimallea** [GitTerraGame/GitTerra#20]
## Please tell us what you DISLIKE about our Game?

I didn't know what to expect when I got to the homepage (I clicked on the link in Sergey's tweet and went straight to webpage). Is this a reference to some other game that I have not played?

## How we can make the Game better?

Provide more information/explanation. Is there a background story to "terra" and generating maps? I feel like I am missing some context. I expected the game to be interactive but I just get a static image generated for me. Is the map generated using my git history? What makes it unique? How do I play?

**Comment by sergeychernyshev** [comment]
Thank you for feedback, @kaimallea!!!

What's the best way to describe the goals of the game? Just write a background story?

And yes, it is not interactive yet - the idea is to make it interactive by helping you make changes to your project and reflecting the change in the map and other things like stats, but maybe add some other interactivity as mini-games on the map or something.

I'd love to hear your ideas though!

**Comment by kaimallea** [comment]
I think maybe what I was expecting was a "marketing" landing page to tell me what it is, and get me excited about what's to come, even though it's not finished yet. I know making the game is a higher priority atm. ;-)

**Comment by sergeychernyshev** [comment]
No, you're absolutely right - all the awesomeness is in our heads, event if it's not going to be fully implemented, it doesn't mean we can't project it onto the user so they are equally excited about it and can start using it or wait till it is more fully featured.

**Comment by sergeychernyshev** [comment]
I added some info about the game to the splash screen. How does that look?
(if you close it, you can click "About Git Terra" in the footer to reopen it)

**Comment by sergeychernyshev** [comment]
Hey @kaimallea, can you try playing it again?
We've built new version that works in GitHub Actions instead and can publish to GitHub Pages.

Would love to know your feedback on how it goes.
